_current_challenges/01_workshops.md

```diff
@@ -7,33 +7,30 @@ excerpt: Details of upcoming SAMPL in-person and virtual workshops, including ag
 permalink: /current/workshops/
 ---
 
+## The 2019 D3R/SAMPL Joint Workshop
+
+The next [face-to-face workshop](https://drugdesigndata.org/about/d3r-2019-workshop) is Aug. 22-23, 2019 in San Diego, joint with D3R.
+Participation is relatively limited but we are checking on whether virtual attendance is a possibility.
+
+# Previous recent workshops
+
 ## The SAMPL6 LogP Virtual Workshop
 
-We have a virtual pre-workshop on the SAMPL6 LogP challenge on Thursday, May 16, beginning at 7 am US Pacific time.  
+We had a virtual pre-workshop on the SAMPL6 LogP challenge on Thursday, May 16, beginning at 7 am US Pacific time.  [SAMPL6 logP virtual workshop video](https://www.youtube.com/watch?v=FWUPXG8U3UE) is now available.
 
 ### Overview/Purpose
 
-The purpose of this workshop is to go over a preliminary evaluation of results, begin considering analysis and lessons learned, and nucleate opportunities for follow up and additional discussion. Part of the goal is to facilitate discussion so that participants can work *together* to maximize lessons learned in the lead up to the August in-person D3R/SAMPL workshop and the JCAMD special issue.
+The purpose of this workshop was to go over a preliminary evaluation of results, begin considering analysis and lessons learned, and nucleate opportunities for follow up and additional discussion. Part of the goal is to facilitate discussion so that participants can work *together* to maximize lessons learned in the lead up to the August in-person D3R/SAMPL workshop and the JCAMD special issue.
 We hope this early workshop will result in additional follow-up afterwards.
 
 ### Agenda
 
-A tentative agenda for this workshop is (all times US Pacific (PDT; UTC-7)):
+The workshop agenda is (all times US Pacific (PDT; UTC-7)):
```

Comment:
"The workshop agenda _is_.." --> "The workshop agenda _was_.."

Reply:
If you suggest a change, don't approve it and merge it; makes it harder to fix!

Reply:
I figured I could merge it and then do a separate pr after, but I had a bunch of other commits and didn't know how to push only one commit. I overcomplicated a single word change  o.o